### PR TITLE
Enhancement updater stability

### PR DESCRIPTION
**Pull request by Atralupus**
- Add confirm update view
- Enhancement stability

### CODE REVIEW

**Review by Akamig — APPROVED** [review]
appropriate timespan for update checking, other part checked, LGTM.

File: src/worker/checkForUpdateWorker.js [hunk 1]
```diff
@@ -24,36 +38,71 @@ setInterval(() => {
...
-              if (file.os === os) {
-                process.send?.({
-                  type: "player update",
-                  path: `${baseUrl}/player/${file.path}`,
-                  size: file.size,
-                });
-              }
-            }
-          }
+          const local = readLocalPlayerVersionFile(err, playerVersionData);
```
Comment (longfin) [inline src/worker/checkForUpdateWorker.js:41]: It also would be better if we can use [`util.promisify()`](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original) for `fs.readFile()`, to replace the callback chain to promise based control flow.

**Review by longfin — APPROVED** [review]

File: src/worker/checkForUpdateWorker.js (hunk 1 above)
Comment (Atralupus) [reply src/worker/checkForUpdateWorker.js]: Thank you. It may be less stable, but I missed it.
I have completed the task and would appreciate your review.

Comment (Atralupus) [reply src/worker/checkForUpdateWorker.js]: I had received approval, so I proceeded to merge the PR. 👀

**Pull request merged**